src/resembleImage.js

```diff
@@ -32,36 +30,41 @@ function resolveFidelity (width, pair) {
     return number;
 }
 
-export default function resembleImage (image, {generator, fidelity}) {
-    return resolver.getResource(image).then(data => data.contents)
-    .then(imageSize).then(({width, height}) => {
-        return new Promise((resolve, reject) => {
-            paper.setup(new paper.Size(width, height));
-            const raster = new paper.Raster(image);
-            const colourStop = colourStopFactory(width);
-
-            raster.onLoad = function () {
-                const stops = [];
-                raster.position = paper.view.center;
-                let chunk;
-                try {
-                    chunk = resolveFidelity(width, fidelity);
-                } catch (e) {
-                    return reject(e);
-                }
-
-                for (let i = 0; i < width; i += chunk) {
-                    const rect = new paper.Path.Rectangle(
-                        i,
-                        0,
-                        chunk,
-                        height
-                    );
-                    stops.push(colourStop(raster.getAverageColor(rect).toCSS(true), i));
-                }
+function rgbToHex ({r, g, b}) {
+    function convert (c) {
+        const hex = c.toString(16);
+        return hex.length === 1 ? '0' + hex : hex;
+    }
+    return convert(r) + convert(g) + convert(b);
```

Comment:
We need a leading `#` for this value; I used to have actual hex values in the tests but the values don't actually align across platforms. But we can update the tests to make sure that each stop is a hex colour of some sort:

```js
function assertColourStops (t, fixture, expected, options) {
    return postcss(plugin(options)).process(fixture).then((result) => {
        valueParser(result.root.first.nodes[0].value).walk(node => {
            if (node.value !== 'linear-gradient') {
                return false;
            }
            const stops = getArguments(node).slice(1);
            t.deepEqual(stops.length, expected);
            stops.forEach(stop => {
                const colour = stop[0].value;
                t.truthy(
                    colour.length === 7 && !isNaN(parseInt(colour.slice(1), 16))
                );
            });
        });
    });
}
```